### PR TITLE
Merge duplicate Node.js tags (Issue #637)

### PR DESCRIPTION
**Pull request by nico-bachner**

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -2427,7 +2427,7 @@ module.exports = [
       'Developer',
       'Software Engineer',
       'Full Stack',
-      'nodejs',
+      'node',
```
Comment (davidak) [inline src/data.js:2430]:
```suggestion
      'Node',
```
that would be a duplicate again

**Review by davidak — APPROVED** [review]
looks good

only one typo

**Review by HugoDF — CHANGES_REQUESTED** [review]
Technically "Node.js" is the official spelling https://nodejs.org/en/

**Comment by nico-bachner** [comment]
I used Node because it was the more popular name, but if it's ok, I'd be more than happy to change it to Node.js

**Comment by HugoDF** [comment]
> I used Node because it was the more popular name, but if it's ok, I'd be more than happy to change it to Node.js

I'm not particularly fussy, but if the trademark is on Node.js then we should probably use that 

Also, I think this branch has conflicts that you'll need to resolve.

**Comment by nico-bachner** [comment]
Should be all good now

**Review by davidak — APPROVED** [review]

**Pull request merged**